config/waagent-extn.logrotate

```diff
@@ -0,0 +1,22 @@
+/var/log/azure/*/*.log {
+    # Old versions of log files are compressed with gzip by default.
+    compress
+
+    # Rotate log files > 10 MB, and keep last 20 archived files. With an average compression ratio of around 10:1, The
```

Comment:
> 20 archived files

10?

Reply:
The 20 comes from the Windows agent. They keep 20 files of the compressed rolled-over logs.

Reply:
i meant line 8 says 10

Reply:
(plus the accounting in your comment doesn't work with 20 :))